### PR TITLE
LPS-163069 | Test Automation

### PR DESCRIPTION
**Pull request by Luizcarlosqueiroz**
**Ticket:**
[LPS-163069](https://issues.liferay.com/browse/LPS-163069)

**Test:**
Object#CanFilterObjectEntriesViaAPI

Test Steps:
1. Create a custom object via API
2. Add a text field called "Food" via API
3. Publish
4. Create several entries with the following text:
8. Use the API to search for Banana

### CODE REVIEW

**Comment by Luizcarlosqueiroz** [comment]
ci:test:sf

**Comment by Luizcarlosqueiroz** [comment]
ci:test:objects

**Comment by Luizcarlosqueiroz** [comment]
ci:test:sf

**Comment by Luizcarlosqueiroz** [comment]
ci:test:sf

**Comment by Luizcarlosqueiroz** [comment]
ci:test:sf

**Review by andreefarias — CHANGES_REQUESTED** [review]
Hey @Luizcarlosqueiroz,

Occurs an error when run locally, fix it, please!

**Review by andreefarias — APPROVED** [review]

**Comment by timpak** [comment]
Taking a look

File: modules/apps/object/object-test/src/testFunctional/tests/Object.testcase [hunk 1]
```diff
@@ -300,6 +300,47 @@ definition {
 			value1 = "Input does not match");
 	}
 
+	@description = "Verify it's possible to sort object entries by API"
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/tests/Object.testcase:303]: Typo, I think you mean search here

**Review by timpak — CHANGES_REQUESTED** [review]
@Luizcarlosqueiroz can you also make the same changes as requested in https://github.com/liferay-bpm-qa/liferay-portal/pull/342#pullrequestreview-1286576388 here as well? There's also a small typo in the description.

The Given When Then is going to be slightly different:

Given: An custom object with a text field and three object entries
- Add custom object with field and publish
- Add object entries

When: An object entry is searched for using the API
- Hit the endpoint and store the result in a var

Then: one object entry is returned based on the search parameter

- Assertion only

**Comment by timpak** [comment]
@Luizcarlosqueiroz can you take a look at the requested changes?

**Comment by Luizcarlosqueiroz** [comment]
ci:test:sf

**Comment by Luizcarlosqueiroz** [comment]
ci:forward

**Pull request closed without merging**